### DISCUSSION TITLE
Cant Install latest Jupyter extension due to vscode 1.79 not being released.

### DISCUSSION THREAD

**Discussion by MasterBoyd** [microsoft/vscode-jupyter#13538] [General]
The latest released version of vscode is 1.78 but the minimum version for install of this jupyter extension is 1.79.
"engines":{"vscode":"^1.79.0"}
How can I get this installed?

**Comment by amunger** [comment]
The extensions tab within vs code should install the latest release of jupyter v2023.4.1011241018, and that only requires vscode 1.78.

If you want to use the latest **pre**-release of the jupyter extension, you can use it on [vscode insiders](https://code.visualstudio.com/insiders/).
